api/camunda/camunda-openapi.yaml

```diff
@@ -778,12 +863,300 @@ paths:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
-          description: Internal server error
+          description: Internal Server Error
```

Comment:
change-at-the-source, regression, me:

I think we missed this case change in https://github.com/camunda/camunda-docs/pull/4196.

Reply:
https://github.com/camunda/camunda/pull/22037